### ISSUE TITLE
复现resent20遇到的问题

### ISSUE THREAD

**Issue by zsureuk** [qiulinzhang/SPConv.pytorch#14]
您好，首先非常感谢您能够提出非常优秀的卷积结构。
我尝试着复现您在resnet20的结果，于是修改您分享的SPConv文件以便它能够在cifar数据集上运行，然而我复现得到的resent20的实验结果（90.4%）远小于base model的精度（92.04%）。
我使用了您在论文中提到的训练参数：SGD with weight decay= 5e-4, batch-size = 128, initial learning rate = 0.1 然后每50epochs降低为原来的0.1。
同样在这种情况在resnet56以及ablation study里的实验也是如此。
请问我复现使用的参数跟您之前做的时候比有什么疏漏的地方吗？请问您可否分享一下关于cifar的训练和模型文件呢？
非常期待作者的回复。

**Comment by qiulinzhang** [comment]
Hello，我今天晚上跑了一下是可以复现的啊，我用的就是 https://github.com/kuangliu/pytorch-cifar 这里面的代码，没做其他修改，只是插入了 SPConv3x3 卷积 替换原来的 nn.conv2d(kernel_size=3)

**Comment by qiulinzhang** [comment]
> Hello，我今天晚上跑了一下是可以复现的啊，我用的就是 https://github.com/kuangliu/pytorch-cifar 这里面的代码，没做其他修改，只是插入了 SPConv3x3 卷积 替换原来的 nn.conv2d(kernel_size=3)
